welpo/tabi
# ✨ feat(header): add img_alt attribute for customisable alt text

## Description

This PR introduces a new `img_alt` attribute within the `header` TOML configuration to allow for customisable alt text for the home banner image. Resolves #150.

#### Changes

- Added optional `img_alt` attribute to `header` variable for the main index page. Defaults to "the owner" (previous hardcoded value).
- Modified `home_banner.html` to use `img_alt` for the `alt` attribute if available.

## Why is this change necessary?

The alt text for images is crucial for both SEO and accessibility. By enabling the `img_alt` attribute, we can provide a more descriptive and contextual alt text, which could vary based on the image used. This makes the theme more versatile and user-friendly.

## How to test

1. Update the `header` in the TOML configuration of an `_index.md` file to include `img_alt` attribute like this:
   ```toml
   header = {title = "Hello! I'm Your Name.", img = "img/main.webp", img_alt = "Your Name, the site's author" }
   ```
2. Build the site and verify that the alt text has been updated as specified in `img_alt`.
